File: Java/FizzBuzz.java
public class FizzBuzz{
    public static void fizzBuzz(int n){
        for (int i=1; i<=n; i++){
            if (i % 15 == 0)
                System.out.println("Fizz Buzz");
            else if (i % 3 == 0)
                System.out.println("Fizz");
            else if (i % 5 == 0)
                System.out.println("Buzz");
            else
                System.out.println(i);
        }
    }
    public static void main(String[] args){
        int n = Integer.parseInt(args[0]);
        fizzBuzz(n);
    }
}

File: Julia/fizzbuzz.jl
#!/usr/bin/julia

function fizzbuzz(n::Int)
    for i = 1 : n
        if i % 15 == 0
            println("Fizz Buzz")
        elseif i % 3 == 0
            println("Fizz")
        elseif i % 5 == 0
            println("Buzz")
        else
            println(i)
        end
    end
end

fizzbuzz(parseint(ARGS[1]))
